configure

```diff
@@ -936,6 +938,15 @@ do
   | -silent | --silent | --silen | --sile | --sil)
     silent=yes ;;
 
+  -runstatedir | --runstatedir | --runstatedi | --runstated \
```

Comment:
What parameters are you supplying when running `autoconf` to generate this portion?

Reply:
None. I just say `autoconf`.

Reply:
And it is 2.69 if that matters:

```sh
edd@rob:~/git/rcpparmadillo(master)$ autoconf --version
autoconf (GNU Autoconf) 2.69
Copyright (C) 2012 Free Software Foundation, Inc.
License GPLv3+/Autoconf: GNU GPL version 3 or later
<http://gnu.org/licenses/gpl.html>, <http://gnu.org/licenses/exceptions.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by David J. MacKenzie and Akim Demaille.
edd@rob:~/git/rcpparmadillo(master)$ 
```